pkg/status/status.go

```diff
@@ -345,91 +324,13 @@ func getCommonStatus() (map[string]interface{}, error) {
 	stats["build_arch"] = runtime.GOARCH
 	now := time.Now()
 	stats["time_nano"] = now.UnixNano()
+	stats["netflowStats"] = netflowServer.GetStatus()
```

Comment:
I extracted this outside of `expvarStats` because is not using expvar to report the information